index.d.ts

```diff
@@ -7,12 +7,16 @@ export function client(url: string, options?: Options): Client;
 export function over(socketType: any, options?: Options): Client;
 
 export class Client {
-  connected:boolean;
+  connected: boolean;
+  isBinary: boolean;
+  partialData: string;
+  subscriptions: SubscriptionsMap;
+  ws: any;
```

Comment:
`ws` is of type `WebSocket`

Reply:
I used `any` because the function `over` receives a `socketType` also as `any`.
And this can have conflicts with no native WebSockets and SockJS.